src/Core/x/xVec3.cpp

```diff
@@ -9,7 +9,12 @@
 #pragma GLOBAL_ASM("asm/Core/x/xVec3.s", "xVec3NormalizeFast__FP5xVec3PC5xVec3")
 
 // func_8004E4DC
-#pragma GLOBAL_ASM("asm/Core/x/xVec3.s", "xVec3Copy__FP5xVec3PC5xVec3")
-
+//#pragma GLOBAL_ASM("asm/Core/x/xVec3.s", "xVec3Copy__FP5xVec3PC5xVec3")
+void xVec3Copy(xVec3 * o, const xVec3 * a){
+    float tmp1[] = {((float*)(void*)a)[0],((float*)(void*)a)[1]};
+    float tmp2[] = {((float*)(void*)a)[2],((float*)(void*)a)[3]};
+    *o = *(xVec3*)tmp1;
+    *(float*)(o+0x8)= tmp2[0];
+}
```

Comment:
This doesn't 🆗 right? I can't merge this code if it's uncommented and doesn't build properly. Also, as discussed in discord, this function was probably hand-written assembly, so I highly doubt that's anything close to what they wrote